lib/holdings/item.rb

```diff
@@ -114,8 +114,13 @@ def bound_with_parent
       item_display[:bound_with]
     end
 
-    # @return [Bool] true if this is a bound-with child
+    # @return [Bool] true if this is a bound-with
     def bound_with?
+      bound_with_child? || bound_with_principal?
```

Comment:
Do you also need to change the caller in `SearchResult::LocationComponent`?

Reply:
My understanding was that this PR is just renaming and not affecting functionality.  Is that incorrect?  There is no linked issue or explanation in the commit message to indicate otherwise.

Reply:
okay I can do that.